cmd/pktvisord/main.cpp

```diff
@@ -571,7 +572,17 @@ int main(int argc, char *argv[])
         exit(EXIT_FAILURE);
     }
 
+    std::string iface;
     if (args["IFACE"]) {
+        iface = args["IFACE"].asString();
+    } else {
```

Comment:
so I don't think we should automatically create a pcap policy if no arguments are passed, because by default i want the user to specify --admin-api or an interface still (which is only for legacy reasons). so, i think we should only do this interface selection and the legacy pcap policy if they pass `auto` as the interface.

however - i do think we can automatically create `default_pcap` tap on the most used interface _if_ there is no `default_pcap` in the config file.

Reply:
But we should lower the barrier Right? be able to execute pktvisord without any value...
The implementation was wrong before because for any case the default policy was being created. I've changed it to:
 It should only should find and create interface if no interface was specified AND config file was not set (orb agent always set a config file for instance)
 
 What do you think? Pr is updated with the approach that I mentioned